customer.go

```diff
@@ -4,22 +4,23 @@ import "time"
 
 // CustomerAttributes are the attributes of a lemonsqueezy customer
 type CustomerAttributes struct {
-	StoreID                       int       `json:"store_id"`
-	Name                          string    `json:"name"`
-	Email                         string    `json:"email"`
-	Status                        string    `json:"status"`
-	City                          *string   `json:"city"`
-	Region                        *string   `json:"region"`
-	Country                       string    `json:"country"`
-	TotalRevenueCurrency          int       `json:"total_revenue_currency"`
-	Mrr                           int       `json:"mrr"`
-	StatusFormatted               string    `json:"status_formatted"`
-	CountryFormatted              string    `json:"country_formatted"`
-	TotalRevenueCurrencyFormatted string    `json:"total_revenue_currency_formatted"`
-	MrrFormatted                  string    `json:"mrr_formatted"`
-	CreatedAt                     time.Time `json:"created_at"`
-	UpdatedAt                     time.Time `json:"updated_at"`
-	TestMode                      bool      `json:"test_mode"`
+	StoreID                       int               `json:"store_id"`
+	Name                          string            `json:"name"`
+	Email                         string            `json:"email"`
+	Status                        string            `json:"status"`
+	City                          *string           `json:"city"`
+	Region                        *string           `json:"region"`
+	Country                       string            `json:"country"`
+	TotalRevenueCurrency          int               `json:"total_revenue_currency"`
+	Mrr                           int               `json:"mrr"`
+	StatusFormatted               string            `json:"status_formatted"`
+	CountryFormatted              string            `json:"country_formatted"`
+	TotalRevenueCurrencyFormatted string            `json:"total_revenue_currency_formatted"`
+	MrrFormatted                  string            `json:"mrr_formatted"`
+	Urls                          map[string]string `json:"urls"`
```

Comment:
You can use an inline struct like this https://github.com/NdoleStudio/lemonsqueezy-go/blob/9a405ae7a658cacbe5a0393440fbffa965b57a5e/order.go#L33 to represent the URL.